esmvaltool/recipes/recipe_OceanBGC.yml

```diff
@@ -0,0 +1,117 @@
+    biogeochemistry. Please use this file as a template for adding additional
+    fields into the ocean. This work based on the BGC-val toolkit GMD-2018-103. 
+    Written by Lee de Mora, Plymouth Marine Laboratory
+    
+  authors:
+    - ledm
+
+  references:
+    - BGC-val:gmd-2018-103
+
+  projects:
+    - ukesm
+    
+
+datasets:
+# working datasets
+  - {dataset: CanESM2,       project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 2001,  end_year: 2004}
+#  - {dataset: GISS-E2-H,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: HadGEM2-AO,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: HadCM3,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: CanCM4,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: CSIRO-Mk3-6-0,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+
+#####
+# datasets with Errors (right now):
+
+# Problem with latitude > 90.
+#  - {dataset: HadGEM2-CC,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004} 
+#  - {dataset: HadGEM2-ES,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004} 
+
+  
+###
+# Problem with times
+#  - {dataset: MIROC-ESM,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: MIROC-ESM-CHEM,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+
+# Unstructured grids                 
+#  - {dataset: MPI-ESM-LR,  project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2003}
+#  - {dataset: MPI-ESM-MR,  project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: ACCESS1-0,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}  
+#  - {dataset: ACCESS1-3,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+
+  
+  
+preprocessors:
+  # --------------------------------------------------
+  # Depth-profile preprocessors
+  # --------------------------------------------------
+  prep_depth_integration_1: # For a 2D map global profile
+    fix_data:     
+    fix_metadata: 
+    depth_integration:
+      coordz: depth
+#      new_units: kg m-2 # need to specify in advance, as cf_units has strange behaviour.
+    time_average:  
+    
+
+
+  # --------------------------------------------------
+  # map preprocessors
+  # --------------------------------------------------
+  prep_surface_map_1: # For a 2D global surface map
+    fix_data:     
```

Comment:
Same here?

Reply:
Can you be more specific as to what the problem is, please?

Reply:
Do you need the empty `fix_data` and `fix_metadata` dicts? Fixes should be applied fully automatically by the preprocessor and there should be no need to specify them here.